packages/language/test/linking.test.ts

```diff
@@ -181,17 +243,119 @@ describe("Linking tests", () => {
 
  PUT(A.<|b>B);`));
 
-    /**
-     * TODO: Implement star handling in structured names
-     */
-    test.skip("Star name in structure should be qualifiable", () =>
+    test.skip("Star name in structure should result in partial qualification", () =>
```

Comment:
Same here, would be good to add a WillFix and a very short description of the limitation